mendix/widgets-resources
# feat(datagrid-web): implementing state for sorting, filtering, hiding, paging and reordering

In this PR I am:

## Features

- Implementing state storing for initial state in sorting, filtering, hiding, paging and reordering
- Implementing filter method (startsWith, endsWith and contains)
- Fixing CSS when columns contains big untruncated texts
- Making it ready to receive custom configuration object from the users/SP
- Renaming properties not complying with Mendix standards

## How to test ?
- Add a column with dynamic custom with a Button that open a popup page receiving a context parameter, then add a datagrid in the new page with editable fields. When you save the Data it should keep the current state of the datagrid. If tested with previous versions it will reset the DG as soon as you save or cancel in the popup. 
- Use variations: Microflow & nanoflow action buttons that change a parameter of a received object
